### PR TITLE
LPS-178371 - Fix Automation | Objects | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
Ticket:
[LPS-178371](https://issues.liferay.com/browse/LPS-178371)

Failing test:
- ObjectCustomLayouts#CanAddEntryOnRelationshipTabForMtoM 
- ObjectCustomLayouts#CanSelectExistingEntriesOnRelationshipTab

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase [hunk 1]
```diff
@@ -1106,13 +1106,11 @@ definition {
...
-				key_titleFieldName = "Entry Test C",
-				locator1 = "CreateObject#SELECT_ENTRY_IN_RELATIONSHIP_TAB");
+			SelectFrame(value1 = "relative=top");
 
-			WaitForElementPresent(
-				key_fieldLabel = "Entry Test C",
-				locator1 = "ObjectAdmin#VIEW_OBJECT_FIELD_LABEL");
+			Click.clickNoMouseOver(locator1 = "Modal#CLOSE_BUTTON");
+
+			CreateObject.addRelationshipAndSelectEntry(titleFieldName = "Entry Test C");
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase:1115]: @thaiscabrals hmm I don't think we need to repeat the same steps to make sure `Entry Test C` is there. Would just selecting the frame and then waiting for the page to load work just like in `CreateObject.addRelationshipAndSelectEntry`?

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Hi @thaiscabrals! Thank you for the fix. I left a comment but please let me know what you think. Thank you!

For next time, the index.html file is what you would want to upload in JIRA so we can view the passing test/s. It should be under the folder of the test in the `test-results` folder. Just make sure you also rename the file to the test especially if there are more than one test you are uploading so it is easier to distinguish. Thank you!

**Comment by thaiscabrals** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase [hunk 2]
```diff
@@ -1097,6 +1099,8 @@ definition {
 				locator1 = "ObjectAdmin#KEBAB_MENU_OPTION");
 
 			SelectFrame(locator1 = "IFrame#MODAL_BODY");
+
+			WaitForPageLoad();
```
Comment (patriciajeanperez) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase:1103]: I think this should be after you click on Entry Test C? @thaiscabrals

**Comment by thaiscabrals** [comment]
ci:test:sf

**Pull request closed without merging**